src/gui/properties/speedplotview.cpp

```diff
@@ -276,9 +281,15 @@ quint64 SpeedPlotView::maxYValue()
         if (!m_properties[static_cast<GraphID>(id)].enable)
             continue;
 
-        for (int i = static_cast<int>(queue.size()) - 1, j = 0; (i >= 0) && (j < m_viewablePointsCount); --i, ++j)
-            if (queue[i].y[id] > maxYValue)
-                maxYValue = queue[i].y[id];
+        milliseconds duration {};
```

Comment:
`milliseconds::zero()`

Reply:
https://github.com/qbittorrent/qBittorrent/pull/14054#discussion_r556293478

Reply:
>`milliseconds::zero()`

Maybe `0ms`? `milliseconds::zero()` is too long... Or `auto duration = milliseconds::zero();`.

Reply:
I will prefer 0ms.